rcsb/utils/dictionary/DictMethodEntityHelper.py

```diff
@@ -736,34 +737,39 @@ def filterSourceOrganismDetails(self, dataContainer, catName, **kwargs):
                             cObj.setValue(";".join([provSource for jj in range(len(tgL))]), "rcsb_gene_name_provenance_source", iRow)
                         else:
                             cObj.setValue(v[ii], at, iRow)
-                        # if at == 'ncbi_taxonomy_id' and v[ii] and v[ii] not in ['.', '?'] and v[ii].isdigit():
                         if at == "ncbi_taxonomy_id" and v[ii] and v[ii] not in [".", "?"]:
-                            taxId = int(self.__reNonDigit.sub("", v[ii]))
-                            taxId = taxU.getMergedTaxId(taxId)
-                            cObj.setValue(str(taxId), "ncbi_taxonomy_id", iRow)
-                            entryTaxIdD[taxId] += 1
-                            entityTaxIdD.setdefault(entityId, set()).add(taxId)
-                            #
-                            sn = taxU.getScientificName(taxId)
-                            if sn:
-                                cObj.setValue(sn, "ncbi_scientific_name", iRow)
-                            #
-                            psn = taxU.getParentScientificName(taxId)
-                            if psn:
-                                cObj.setValue(psn, "ncbi_parent_scientific_name", iRow)
-                            #
-                            cnL = taxU.getCommonNames(taxId)
-                            if cnL:
-                                fcnL = self.__filterCaseDuplicates(cnL)
-                                cObj.setValue(";".join(list(OrderedDict.fromkeys(fcnL))), "ncbi_common_names", iRow)
-                            # Add lineage -
-                            linL = taxU.getLineageWithNames(taxId)
-                            if linL is not None:
-                                cObj.setValue(";".join([str(tup[0]) for tup in OrderedDict.fromkeys(linL)]), "taxonomy_lineage_depth", iRow)
-                                cObj.setValue(";".join([str(tup[1]) for tup in OrderedDict.fromkeys(linL)]), "taxonomy_lineage_id", iRow)
-                                cObj.setValue(";".join([str(tup[2]) for tup in OrderedDict.fromkeys(linL)]), "taxonomy_lineage_name", iRow)
+                            if not v[ii].strip().isdigit():
+                                logger.warning("Source attribute 'ncbi_taxonomy_id' value is not integer: %r. Will attempt to sanitize.", v[ii])
```

Comment:
Perhaps add `entry_id` to the warning and error messages here and in other places?